NOnion/Utility/HSUtility.fs

```diff
@@ -72,3 +72,16 @@ module HSUtility =
             GetStartTimeOfNextTimePeriod srvStartTime hsDirInterval
 
         not(now >= srvStartTime && now < tpStartTime)
+
+    // https://github.com/torproject/torspec/blob/cb4ae84a20793a00f35a70aad5df47d4e4c7da7c/rend-spec-v3.txt#L2161
+    let GetPublicKeyFromUrl(url: string) =
+        //Remove .onion suffix and decode
+        let keyBytesOpt =
+            url.Split('.') |> Seq.tryHead |> Option.map Base32Util.DecodeBase32
```

Comment:
no need for parens here

Reply:
Fixed in [aae9fe3](https://github.com/nblockchain/NOnion/pull/30/commits/aae9fe32759d73c848bad5769597077cf7026598)

Reply:
This should be fixed in this commit, not in other one. But if you're too lazy to do that, at least do it in a refactoring commit so that it can be added to .git-blame-ignore-revs